.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "submodules/privacy-grade"]
 	path = submodules/privacy-grade
 	url = https://github.com/duckduckgo/privacy-grade
+[submodule "DuckDuckGoTests/MockFiles/reference-tests"]
```

Comment:
Please put submodules under the submodules path

Reply:
I tried this approach first, but being a Swift nub I couldn't figure out how to map that dir to a resource bundle.  This submodule just contains JSON-based test definitions, no code.

If you have any pointers on how to do that I'd definitely appreciate it!

Reply:
Fixed!